lib/chef/resource/apt_package.rb

```diff
@@ -75,6 +84,9 @@ class AptPackage < Chef::Resource::Package
         description: "A Hash of response file variables in the form of {'VARIABLE' => 'VALUE'}.",
         default: {}, desired_state: false
 
+      property :anchor_package_name_patterns, [TrueClass, FalseClass],
+        description: "A Boolean flag to decide whether packages which have pattern like names should be regex matched.",
```

Comment:
Can you add a comment here to describe the regex match as a strict or explicit match of the name as typed to make that clearer?

Reply:
More details added in the example https://github.com/chef/chef-web-docs/pull/4156/files#diff-ebf5d0923677843ee47e426717385ec01b3d788bd19a37a29c8ed3a42cc6df4eR148